persistent/Database/Persist/Sql/Orphan/PersistStore.hs

```diff
@@ -41,27 +41,11 @@ import Web.HttpApiData (ToHttpApiData, FromHttpApiData)
 import Database.Persist
 import Database.Persist.Class ()
 import Database.Persist.Sql.Class (PersistFieldSql)
-import Database.Persist.Sql.Internal (generatedAttribute)
 import Database.Persist.Sql.Raw
 import Database.Persist.Sql.Types
 import Database.Persist.Sql.Util (
     dbIdColumns, keyAndEntityColumnNames, parseEntityValues, entityColumnNames
-  , updatePersistValue, mkUpdateText, commaSeparated)
-
--- Contributor note: any time you execute a prepared statement to
--- insert one or more rows, you need to use 'valuesToInsert entity' to
--- generate the list of values that you feed into your prepared
--- statement so that generated columns are properly scrubbed.
-valuesToInsert :: PersistEntity entity => entity -> [PersistValue]
```

Comment:
I'm glad you changed this, because I just realized it introduces a bug in the Sqlite backend where adding a `generated=` attribute to a field (correctly) doesn't make the field generated in the schema but (incorrectly) redacts the value when binding to a prepared statement <.<;